src/amulware.Graphics/Core/shading/VertexAttribute.cs

```diff
@@ -40,14 +38,13 @@ sealed public class VertexAttribute
         /// Sets the attribute for a specific program.
         /// </summary>
         /// <param name="program">The program.</param>
-        public void setAttribute(ShaderProgram program)
+        public void SetAttribute(ShaderProgram program)
```

Comment:
Isn't this being used anywhere?

Reply:
Not in this library, since I can build the project just fine. It may be used outside, which is why this is a breaking change as explained in the PR description.